src/calculations.cpp

```diff
@@ -79,12 +80,12 @@ const shared_ptr<Output> activateOutputs(const list<shared_ptr<Output>> &outputs
     return primary;
 }
 
-void ltrOutputs(const list<shared_ptr<Output>> &outputs) {
+void ltrOutputs(const list<shared_ptr<Output>> &outputs, const map<string, string> &copies) {
     int xpos = 0;
     int ypos = 0;
     for (const auto &output : outputs) {
 
-        if (output->desiredActive) {
+        if (output->desiredActive && copies.find(output->name) == copies.end()) {
```

Comment:
We need to test whether the destination (and source) exists. `xlayoutdisplay --copy HDMI1:aoeu` results in HDMI1 being disabled.

A good place might be after `discoverOutputs`, with an error/exit message like "copy src HDMI2222 not found"